geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -319,6 +319,9 @@ pub enum SlotStatus {
 
     /// The highest slot that has been voted on by supermajority of the cluster, ie. is confirmed.
     Confirmed,
+
+    /// First Shred Received
```

Comment:
Are the comments for the other SlotStatus enums wrong? Almost seems like they were copy pasted from confirmation levels.. We send theses notifications out each time a slot transitions to the new state, right (assuming it makes it to that state)?

Reply:
I think the part explaining this is "The highest slot" makes more sense in the context of confirmation levels.

I would probably rewrite them like this:
```rust
    /// First shred has been received for this slot.
    FirstShredReceived,

    /// This slot has been processed.
    Processed,

    /// This slot has been optimistically confirmed by a supermajority of the cluster.
    Confirmed,

    /// This slot has been confirmed and reached max vote lockout on the canonical fork.
    Rooted,
```
I acknowledge this is sort of out of scope for this PR - I just noticed it because your comment is written more like how I would expect for a slot status notification but doesn't quite match the others.